deepmd/pt/utils/stat.py

```
@@ -35,6 +35,19 @@
+# Re-export from dpmodel (backend-agnostic implementations)
+from deepmd.dpmodel.utils.stat import (  # noqa: E402
+    _restore_observed_type_from_file,
+    _save_observed_type_to_file,
+    collect_observed_types,
+)
+
+__all__ = [
+    "collect_observed_types",
+    "_restore_observed_type_from_file",
+    "_save_observed_type_to_file",
+]
```

Ruff (0.15.2)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)